examples/kubernetes/dynamic_provisioning_s3/README.md

```diff
@@ -23,7 +23,7 @@ parameters:
 * autoImportPolicy - the policy FSx will follow that determines how the filesystem is automatically updated with changes made in the linked data repository. For a list of acceptable policies, please view the official FSx for Lustre documentation: https://docs.aws.amazon.com/fsx/latest/APIReference/API_CreateFileSystemLustreConfiguration.html
 * s3ImportPath(Optional) - S3 data repository you want to copy from S3 to persistent volume.
 * s3ExportPath(Optional) - S3 data repository you want to export new or modified files from persistent volume to S3.
-* deploymentType (Optional) - FSx for Lustre supports three deployment types, SCRATCH_1, SCRATCH_2 and PERSISTENT_1. Default: SCRATCH_1.
+* deploymentType (Optional) - FSx for Lustre supports four deployment types, SCRATCH_1, SCRATCH_2, PERSISTENT_1 and Persistent_2. Default: SCRATCH_1.
```

Comment:
Please change formatting Persistent_2 to align with other deployment types (PERSISTENT_2)